modules/ROOT/pages/apikit-tutorial-jsonplaceholder.adoc

```diff
@@ -4,42 +4,31 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: api, raml, json
 
-Using Anypoint Studio 6.x, you can use APIkit to generate a flow based on a API specification written in RAML or WSDL. In this procedure, you use a RAML specification and add a new service to the interface. You call the API to access services. The interface connects to a free online REST service, JSONPlaceholder.  Resources are:
+Using Anypoint Studio 6.x, you can use APIkit to generate a flow based on an API specification written in RAML or WSDL.
```

Comment:
Using Anypoint Studio 6.x, > **In** Anypoint Studio 6.x you can use...

I think this sounds better, just a minor comment.